### PR TITLE
allow running all tests in test target

### PR DESCRIPTION
**Pull request by Germandrummer92**
result in playwright UI image

### CODE REVIEW

File: tests/debugtopus.spec.ts [hunk 1]
```diff
@@ -34,76 +44,114 @@ describe("prepareTestRun", () => {
...
+      const config = getConfig("doesn't/matter", brokenWindowsOutputPath);
 
-    expect(config).toContain('outputDir: "some\\\\broken\\\\path\\\\1234"');
+      expect(config).toContain('outputDir: "some\\\\broken\\\\path\\\\1234"');
+    });
   });
 
-  it("should not have \\ in command path", async () => {
-    const backslashPath = "C:\\User\\Some\\Path";
+  describe(createPlaywrightCommand.name, () => {
```
Comment (Germandrummer92) [inline tests/debugtopus.spec.ts:145]: tests here are the same aside from where I commented, they are just moved into "describes" for each method

File: tests/debugtopus.spec.ts [hunk 2]
```diff
@@ -34,76 +44,114 @@ describe("prepareTestRun", () => {
...
-      code: testCode,
-    });
-    const testFileContent = readFileSync(testFilePath, { encoding: "utf-8" });
-    expect(testFileContent).toEqual(testCode);
-
-    const configFileContent = readFileSync(configFilePath, {
-      encoding: "utf-8",
-    });
-    expect(configFileContent).toEqual(getConfig(url, outputDir));
+  describe(prepareTestRun.name, () => {
```
Comment (Germandrummer92) [inline tests/debugtopus.spec.ts:47]: relevant describe

File: tests/octomind-api.spec.ts [hunk 3]
```diff
@@ -1,54 +1,112 @@
...
+        }),
+      ).rejects.toMatchInlineSnapshot(
+        `[Error: failed to get code from https://app.octomind.dev/api/bearer/v1/test-targets/testTargetId/test-cases/testId/code?executionUrl=https://thisIsARealUrl.com: response body: '"Internal Server Error"' statusCode: 'undefined']`,
+      );
+    });
   });
 
-  it("correctly throws the error", async () => {
-    const axiosError = new AxiosError("some error");
+  describe(getTestCases.name, () => {
```
Comment (Germandrummer92) [inline tests/octomind-api.spec.ts:64]: new

File: tsconfig.json [hunk 4]
```diff
@@ -7,7 +7,7 @@
     "target": "ESNext",
     "module": "NodeNext",
     "outDir": "./dist",
-    "rootDir": "./src",
+    "rootDir": ".",
```
Comment (Germandrummer92) [inline tsconfig.json:10]: required since we now have a file in tests that we import (the mocks.ts)

File: tests/debugtopus.spec.ts [hunk 5]
```diff
@@ -13,17 +13,27 @@ jest.mock("fs", () => ({
   existsSync: jest.fn(jest.requireActual("fs").existsSync),
 }));
 
-describe("prepareTestRun", () => {
+jest.mock("");
```
Comment (fabianhinz) [inline tests/debugtopus.spec.ts:16]: 😱

**Review by fabianhinz — APPROVED** [review]
lgtm, please remove the empty mock

**Pull request merged**